src/common/i18n/translations/zh.ts

```diff
@@ -1,6 +1,10 @@
 import { Translations } from "./type";
 
 export const zh: Translations = {
+  blockUser: {
+    header: "现已在网络上提供",
+    body: "切换到使用最新版本的 SupplyAlly app。",
```

Comment:
I see the usage of `SupplyAlly` here; it should be `GovSupply`. Has the copy here been verified by UX?

Reply:
Oh I have not ask for verification from UX yet 😅